inc/Engine/Optimization/RUCSS/Warmup/Subscriber.php

```diff
@@ -87,6 +100,76 @@ public function start_scanner( $old_value, $value ) {
 		$this->scanner->start_scanner( $old_value, $value );
 	}
 
+	/**
+	 * Checks the warmup status for resources in the prewarmup process
+	 *
+	 * @since 3.9
+	 *
+	 * @return void
+	 */
+	public function check_warmup_status() {
+		if ( ! (bool) $this->options->get( 'remove_unused_css', 0 ) ) {
+			return;
+		}
+
+		$this->status_checker->check_warmup_status();
+	}
+
+	/**
+	 * Displays the prewarmup result notice
+	 *
+	 * @since 3.9
+	 *
+	 * @return void
+	 */
+	public function prewarmup_result_notice() {
+		if ( ! current_user_can( 'rocket_manage_options' ) ) {
+			return;
+		}
+
+		$screen = get_current_screen();
+
+		if (
+			! isset( $screen->id )
+			||
+			'settings_page_wprocket' !== $screen->id
+		) {
+			return;
+		}
+
+		$notice = get_transient( 'rocket_rucss_prewarmup_notice' );
+
+		if ( ! $notice ) {
+			return;
+		}
+
+		rocket_notice_html(
+			$notice
+		);
```

Comment:
I think we need to delete the transient here, right?
or you will let it deleted after this one hour of its lifespan?